.gitignore

```diff
@@ -54,3 +54,17 @@ _test
 .DS_Store
 .env
 scripts/development/
+
+build/.idea/.idea.build.dir/.idea/encodings.xml
+
+build/.idea/.idea.build.dir/.idea/indexLayout.xml
+
+build/.idea/.idea.build.dir/.idea/projectSettingsUpdater.xml
+
+build/.idea/.idea.build.dir/.idea/vcs.xml
+
+build/.idea/.idea.build.dir/.idea/workspace.xml
+
+installer/.idea/.idea.Installer/.idea/projectSettingsUpdater.xml
+
+installer/.idea/.idea.Installer/.idea/workspace.xml
```

Comment:
I'm nitpicking a bit now, but could you please update these lines to this:

`**/.idea/*`

That should exclude all the same files, and ensure that any new `.idea` files are also excluded. Better double-check that the the ignore still works correctly though, I'm not :100: about the syntax